CHANGES_NEXT_RELEASE

```diff
@@ -1,2 +1,3 @@
 - Fix: missing lastSuccess/lastFailure associated to initial notification on subscription creation some times when csub cache is in use (#2974)
+- Fix: several invalid memory accesses (based on a workaround, not a definitive solution) (#2994)
```

Comment:
I prefer the workding in PR #~5:

```
Fix: several invalid memory accesses (based on a workaround, not a definitive solution, see issue #2994)
```

Why? Because with the current wording it seems that 2994 is the issue that has been solved, not the workaround itself

Reply:
Fixed in 60730ec